man/udevadm.xml

```diff
@@ -564,6 +564,18 @@
             <xi:include href="version-info.xml" xpointer="v251"/>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><option>-i</option></term>
+          <term><option>--include-parents</option></term>
```

Comment:
maybe call this ```--with-parents```?

Reply:
I don't have any specific preferences, include-parents was the suggestion from yuwata. My initial name was definitely the worst of both options :smile: